mri/optimizers/primal_dual.py

```diff
@@ -24,7 +24,7 @@
 from modopt.opt.reweight import cwbReweight
 
 
-def condatvu(gradient_op, linear_op, prox_dual_op, cost_op,
+def condatvu(gradient_op, linear_op, prox_op, cost_op,
```

Comment:
I had to this to prevent a lot of changes while refactoring the code in reconstruct function.
I dont think this could be a problem as it is internal at this point. Please let me know if it will be a problem and in which case, could you please suggest what I could do?

Reply:
The best thing to do is to provide a good documentation of this operator